rewrites.js

```diff
@@ -269,7 +269,7 @@ exports.REWRITES = [
   },
   {
     source: '/discover',
-    destination: '/discover',
+    destination: '/search',
```

Comment:
This will preserve the `/discover` URL when you visit the page. Canonical URLs are always better, so I'd suggest to rather remove this route and add a permanent redirect in next config

Reply:
Done. 👍🏽